Farias123/resolveSistemas
# Feature/ft 01 split methods add unit tests

**What does this pr do?**

This pr splits some methods in a new file and add an extra file specific to write unit tests.

**Notes**

all special characters were removed due to ASCII pattern, keeping those characters would implicate some future issues incluiding stopping the unit tests.

**How to test it**

 - On your computer run git fetch on terminal;
 - Verify that the branch `feature/FT-01_split_methods_add_unit_tests` shows up;
 - Checkout the branch `feature/FT-01_split_methods_add_unit_tests`;
 - On terminal run the comand `pip install pytest`;
 - On terminal access your project folder, where the file `test_gauss.py` is located;
 - Run the command `pytest test_gauss.py` and checkout the green light for the existent unit tests.
